org-ql-view.el

```diff
@@ -1032,54 +1032,91 @@ current buffer.  Otherwise BUFFERS-FILES is returned unchanged."
                                (string (expand-file-name it))
                                (otherwise it))
                              list)))
-    ;; TODO: Test this more exhaustively.
-    (pcase buffers-files
-      ((pred listp)
-       (pcase (expand-files buffers-files)
-         ((pred (seq-set-equal-p (mapcar #'expand-file-name (org-agenda-files))))
-          "org-agenda-files")
-         ((and (guard (file-exists-p org-directory))
-               (pred (seq-set-equal-p (org-ql-search-directories-files
-                                       :directories (list org-directory)))))
-          "org-directory")
-         (_ buffers-files)))
-      ((pred (equal (current-buffer)))
-       "buffer")
-      ((or 'org-agenda-files '(function org-agenda-files))
-       "org-agenda-files")
-      ((and (pred bufferp) (guard (buffer-file-name buffers-files)))
-       (buffer-file-name buffers-files))
-      (_ buffers-files))))
+    (-->
```

Comment:
Wrapping a large form like this in `-->` is hard to read.  Please do something like bind the value and then make the `-->` form as small as possible.